buildSrc/build.gradle.kts

```diff
@@ -12,5 +12,5 @@ repositories {
 
 dependencies {
     implementation("com.google.cloud.tools:jib-gradle-plugin:3.3.2")
-    implementation("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:4.2.1.3168")
+    implementation("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:4.3.0.3225")
```

Comment:
We can't upgrade this yet: https://github.com/ministryofjustice/hmpps-probation-integration-services/pull/1967#issuecomment-1660256525